src/Formats/EscapingRuleUtils.cpp

```diff
@@ -1,14 +1,14 @@
-#include <Formats/EscapingRuleUtils.h>
-#include <Formats/SchemaInferenceUtils.h>
-#include <DataTypes/Serializations/SerializationNullable.h>
-#include <DataTypes/DataTypeString.h>
-#include <DataTypes/DataTypeNullable.h>
 #include <DataTypes/DataTypeFactory.h>
```

Comment:
As I can see, all changes in this file are style changes and are not related to actual changes. It makes it difficult to read the diff in this PR. Most likely these changes were made by your IDE. Please, remove these changes.